extensions/resource-deployment/src/services/tools/mssqlctlTool.ts

```diff
@@ -13,7 +13,7 @@ export class MSSQLCtlTool implements ITool {
 	}
 
 	get description(): string {
-		return localize('resourceDeployment.MssqlCtlDescription', 'Command-line tool for installing and managing the SQL Server big data cluster');
+		return localize('resourceDeployment.MssqlCtlDescription', 'The command-line tool to manage SQL Server Big Data Cluster');
```

Comment:
Same comment about casing on big data cluster.

Also consider updating to more closely align with wording on the [mssqlctl docs page](https://docs.microsoft.com/en-us/sql/big-data-cluster/deploy-install-mssqlctl?view=sqlallproducts-allversions)

```
mssqlctl is a command-line utility written in Python that enables cluster administrators to bootstrap and manage the big data cluster via REST APIs. 
```